ci/Jenkinsfile.tests-e2e

```diff
@@ -251,9 +251,6 @@ def formatMap(Map data=[:]) {
 }
 
 def getDefaultBuildSource() {
-  if (JOB_NAME ==~ 'status-desktop/qa-automation/prs/.*') {
-    return 'status-desktop/nightly'
-  }
   return ''
 }
```

Comment:
Why do we keep the function at all?

Reply:
i was not sure about that :) I can remove it. seems no big deal